components/layout.js

```diff
@@ -13,30 +13,35 @@ if (typeof window !== 'undefined' && window.__NEXT_DATA__ !== undefined) {
   rehydrate(window.__NEXT_DATA__.ids)
 }
 
-const Wrapper = glamorous.div((props, theme) => ({
-  fontFamily: theme.font.glamorous,
+const Wrapper = glamorous.div((props, {fonts, colors}) => ({
+  fontFamily: fonts.sansserif,
+  backgroundColor: colors.primaryLight,
+  ':after': {
+    content: '""',
+    pointerEvents: 'none',
+    backgroundImage: 'url(/static/images/g-background.svg)',
+    backgroundSize: '800px',
+    backgroundRepeat: 'no-repeat',
+    backgroundPosition: '70% -300px',
+    position: 'absolute',
+    zIndex: 0,
+    top: 0,
+    left: 0,
+    right: 0,
+    bottom: 0
+  }
 }))
 
-const NavWrapper = glamorous.div({
-  display: 'flex',
-  justifyContent: 'space-between',
-  marginTop: 10,
-  marginRight: 20,
-  marginLeft: 20,
-})
-
-export default Layout
-
-function Layout({children}) {
+export default ({children}) => {
   css.insert(baseStyles())
   return (
     <ThemeProvider theme={GlobalStyles}>
       <Wrapper>
-        <NavWrapper>
-          <Nav />
-        </NavWrapper>
-        {children}
-        <Footer />
+        <Div position="relative" zIndex={1}>
```

Comment:
Let's avoid mixing the syntax pattern between inline and using `glamorous` factory. I'd go back to that `NavWrapper` and just update the styles for it. You also don't need t import `Div` from `glamorous`